### ISSUE TITLE
no-member false positive for functions

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#139]
Originally reported by: **BitBucket: [eran_brodet](http://bitbucket.org/eran_brodet)**

---

Functions have unique members (such as func_doc, func_code, func_name etc.), pylint doesn't seem to recognise this and reports a no-member error.

The following code exhibits the problem:

```
#!python

def my_func():
  pass

print my_func.func_doc
```

The reported error is:
_E: 4,6 Function my_func has no 'func_doc' member (no-member)_

Versions:
Pylint version 1.1.0 (Astroid 1.0.1), Python 2.7

Thanks in Advance, Eran.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/139
